src/components/Auth/styles.js

```diff
@@ -79,15 +106,58 @@ export const AutoLoginText = styled.label`
   color: ${(props) => props.theme.colors.gray};
   margin-left: 4px;
   font-size: 12px;
+
+  :after {
+    content: '자동 로그인';
+  }
+
+  @media screen and (max-width: ${(props) => props.theme.deviceSizes.mobileM}) {
+    :after {
+      content: '로그인 상태 유지';
+    }
+  }
 `;
 
 export const OtherOption = styled.div`
-  width: 380px;
   display: flex;
+  width: 380px;
+  margin-top: 16px;
   align-items: center;
   justify-content: center;
-  font-size: 11px;
-  margin-top: 16px;
+
+  @media screen and (max-width: ${(props) => props.theme.deviceSizes.mobileM}) {
+    margin-top: 16.6px;
+
+    a:nth-child(2) {
+      right: -75px;
+      position: relative;
+
+      ::after {
+        content: none;

```

Comment:
일관성을 위해 `''` 로 통일하는건 어떨까요?

```suggestion
        content: '';

```